### PR TITLE
[INLONG][feature][audit] add audit-store module

### LINKED ISSUE #1882: [Feature] add audit-store module of inlong audit feature
### Description

add audit-store module of inlong audit feature

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by baomingyu**
Fixes #1882

### Motivation
dd audit-store module

### CODE REVIEW

File: inlong-audit/audit-store/conf/logback.xml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+-->
+<configuration  scan="true" scanPeriod="60s">
+	<!--日志配置-->
```
Comment (aloyszhang) [inline inlong-audit/audit-store/conf/logback.xml:21]: please remove these annotation

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/store/db/DruidConfig.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        druidDataSource.setTestOnBorrow(properties.isTestOnBorrow());
+        druidDataSource.setTestOnReturn(properties.isTestOnReturn());
+        druidDataSource.setPoolPreparedStatements(properties.isPoolPreparedStatements());
+        druidDataSource.setMaxPoolPreparedStatementPerConnectionSize(properties
+                .getMaxPoolPreparedStatementPerConnectionSize());
+        try {
+            druidDataSource.setFilters(properties.getFilters());
+            druidDataSource.init();
+        } catch (SQLException e) {
+            e.printStackTrace();
```
Comment (healchow) [inline inlong-audit/audit-store/src/main/java/org/apache/inlong/store/db/DruidConfig.java:68]: not print the stack trace to console.

File: inlong-audit/audit-store/sql/audit.sql [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE TABLE `audit_data` (
+  `id` int(32) not null primary key auto_increment,
+  `ip` varchar(32) NOT NULL DEFAULT '',
+  `docker_id` varchar(100) NOT NULL DEFAULT '',
```
Comment (healchow) [inline inlong-audit/audit-store/sql/audit.sql:21]: Add comments for each column.

File: inlong-audit/audit-store/conf/logback.xml [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+			<maxFileSize>1024MB</maxFileSize>
+			<maxHistory>2</maxHistory>
+			<totalSizeCap>50GB</totalSizeCap>
+		</rollingPolicy>
+		<encoder>
+			<pattern>%d{yyy-MM-dd HH:mm:ss.SSS,GMT+8} %p %t - %m%n</pattern>
+			<charset class="java.nio.charset.Charset">UTF-8</charset>
+		</encoder>
+		<filter class="ch.qos.logback.classic.filter.LevelFilter"><!-- 只打印INFO日志 -->
+			<!--过滤 INFO-->
```
Comment (healchow) [inline inlong-audit/audit-store/conf/logback.xml:41]: Replace Chinese with English.

File: inlong-audit/audit-store/conf/logback.xml (hunk 1 above)
Comment (baomingyu) [reply inlong-audit/audit-store/conf/logback.xml]: done

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/store/db/DruidConfig.java (hunk 2 above)
Comment (baomingyu) [reply inlong-audit/audit-store/src/main/java/org/apache/inlong/store/db/DruidConfig.java]: done

File: inlong-audit/audit-store/sql/audit.sql (hunk 3 above)
Comment (baomingyu) [reply inlong-audit/audit-store/sql/audit.sql]: done

File: inlong-audit/audit-store/conf/logback.xml (hunk 4 above)
Comment (baomingyu) [reply inlong-audit/audit-store/conf/logback.xml]: done

**Review by aloyszhang — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**